src/inverseRobot.js

```diff
@@ -7,7 +7,19 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const values = [];
+  const obj = {};
+
+  for (const key in robot) {
+    if (values.includes(robot[key])) {
```

Comment:
Do not create `values` array at all and check for repeated keys with `in` operator or `hasOwnProperty` method directly in your `obj`, instead of `includes`